sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderAsyncClient.java

```diff
@@ -175,8 +178,11 @@
      * @return metadata for a tileset in the TileJSON format.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<MapTileset>> getMapTilesetWithResponse(TilesetID tilesetId) {
-        return this.serviceClient.getMapTilesetWithResponseAsync(tilesetId);
+    public Mono<MapTilesetPrivate> getMapTileset(TilesetID tilesetId) {
```

Comment:
Cannot return MapTilesetPrivate to the public. Here you need to transform it into a public MapTileset that you ahve modified to have the right attributes.